### PR TITLE
MM-24813 Add e2e tests for archive post menu options

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add e2e tests for archive channel post menu options

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28413

### CODE REVIEW

File: e2e/cypress/integration/archived_channel/post_menu_spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        });
+
+        // # Login as test user and visit create channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T1721 Archive channel posts menu should have copy link and reply options', () => {
+        // # click to add a channel description
```
Comment (Willyfrog) [inline e2e/cypress/integration/archived_channel/post_menu_spec.js:27]:
```suggestion
        // # Click to add a channel description
```

**Review by Willyfrog — APPROVED** [review]
minor nitpick, looking good

**Review by saturninoabril — CHANGES_REQUESTED** [review]
@sudheerDev LGTM, except for one request.  Thanks!

File: e2e/cypress/integration/archived_channel/post_menu_spec.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        // # Get the last post for reference of ID
+        cy.getLastPostId().then((postId) => {
+            // # Archive the channel
+            cy.uiArchiveChannel();
+
+            // # Click on post dot menu
+            cy.clickPostDotMenu(postId);
+
+            // * Copy link menu item should be visible
+            cy.get(`#permalink_${postId}`).should('be.visible');
```
[the remark this answers is not in the document]
Comment (sudheerDev) [reply e2e/cypress/integration/archived_channel/post_menu_spec.js]: @saturninoabril Thank you

**Review by saturninoabril — APPROVED** [review]
Thanks @sudheerDev! LGTM.

**Pull request merged**